tests/networkpolicy_test.go

```diff
@@ -104,14 +90,14 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		It("[test_id:1511] should be failed to reach vmia from vmib", func() {
 			By("Connect vmia from vmib")
-			ip := vmia.Status.Interfaces[0].IP
-			assertPingFail(ip, vmib)
+			Expect(vmia.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+			assertPingFailBetweenVMs(vmib, vmia)
 		})
 
 		It("[test_id:1512] should be failed to reach vmib from vmia", func() {
 			By("Connect vmib from vmia")
-			ip := vmib.Status.Interfaces[0].IP
-			assertPingFail(ip, vmia)
+			Expect(vmib.Status.Interfaces[0].IPs).NotTo(BeEmpty())
```

Comment:
ditto

Reply:
lets continue at
https://github.com/kubevirt/kubevirt/pull/4045#discussion_r478245984

(same deal will be at the end for all the 7 tests)